src/test/java/com/gs/crdtools/GeneratorTest.java

```diff
@@ -14,54 +17,56 @@
 
 public class GeneratorTest {
 
+    private static List<YAMLValue> parsedCrd;
+    private static Map<Path, String> sourceCodeFromSpecs;
     public static final Path OUTPUT_DIR = Path.of(
             "src/main/java",
             OUTPUT_PACKAGE.replaceAll("\\.", "/")
     );
 
-    @Test
-    void testParseCrds() throws IOException {
+    @BeforeAll
+    static void setUp() throws IOException {
```

Comment:
I realise that this is subjective, but I dislike setting up the input data in a `@BeforeAll` method. I would much rather have the extra bit of redundancy of having the data that the test operates on be explicitly created at the top of each test method. The reason for this is that it makes sense to optimise for few indirections for the person coming later and working with the test cases